### DISCUSSION TITLE
how to remove ramps on interferograms?

### DISCUSSION THREAD

**Discussion by sincaliz** [gmtsar/gmtsar#911] [Q&A]
Hi everyone! 

I would like to know if there is a way to correct long wavelength fringes on interferograms? 
I am processing stripmap SM3 (10 m) ALOS2 data and I obtained fringes on my phase output that could be removed with a ramp. However, I do not see if there is a function that allows me to do that. I attached coherence and phase plots. 

I would appreciate any help on this regard.

[phasefilt_mask.pdf](https://github.com/gmtsar/gmtsar/files/15147314/phasefilt_mask.pdf)
[phasefilt.pdf](https://github.com/gmtsar/gmtsar/files/15147315/phasefilt.pdf)
[phase.pdf](https://github.com/gmtsar/gmtsar/files/15147316/phase.pdf)
[corr.pdf](https://github.com/gmtsar/gmtsar/files/15147317/corr.pdf)

**Comment by Xiaohua-Eric-Xu** [comment]
This looks very much like ionospheric phase if the orbit file you used are accurate. Try tuning the ionospheric correction parameters and see whether that gives you some correction. The filtering step might be slow though.

**Reply by Xiaohua-Eric-Xu** [reply]
e.g. in the config file
```
# make ionospheric phase corrections using split spectrum method
correct_iono = 1
iono_filt_rng = 3.0
iono_filt_azi = 0.5
iono_dsamp = 2
# 
# set the following parameter to skip ionospheric phase estimation
iono_skip_est = 0 
```

**Reply by sincaliz** [reply]
Thanks for your reply. 
I am running the ionospheric corrections but as you said it is taking time. In the meantime, could you tell me which is the criteria to select the values for iono_filt_rng, iono_filt_azi and iono_dsamp? Do these parameter change depending on the satellite and mode of acquisition?

**Reply by Xiaohua-Eric-Xu** [reply]
These are mostly emprical. The best way still remains to examine whether the solved ionospheric correction matches what you desire. A stronger filter will for sure smear short wavelength signals but will be less biased by noise. It is dependent on satellites. Personally I feel that the noise in ScanSAR mode is stronger so I would increase the filtering a bit more.
